test/unit/dialects/postgres/query-generator.test.js

```diff
@@ -1071,6 +1071,28 @@ if (dialect.startsWith('postgres')) {
         },
       ],
 
+      deleteQuery: [
```

Comment:
Prefer adding these tests to `test/unit/query-generator/delete-query.test.ts` instead.
We're working on removing dialect specific tests. Instead we want to test the behavior on all dialects. 
In this case, all dialects should expect an error to be thrown, except PG